src/Analysis/Ast/Impl/Specializations/BuiltinsSpecializations.cs

```diff
@@ -91,6 +92,23 @@ public static IMember List(IPythonInterpreter interpreter, IPythonFunctionOverlo
             return args.Count > 0 && args[0] is PythonCollection c ? c.Contents.FirstOrDefault() : null;
         }
 
+        public static IMember Super(IPythonModule declaringModule, IPythonFunctionOverload overload, IArgumentSet argSet, IndexSpan indexSpan) {
+            foreach (var s in argSet.Eval.CurrentScope.EnumerateTowardsGlobal) {
+                if (s.Node is ClassDefinition) {
+                    var classType = s.Variables["__class__"].GetPythonType<IPythonClassType>();
+
+                    if (classType?.Mro != null) {
+                        var proxySuper = new PythonSuperType(new Location(declaringModule, indexSpan), classType.Mro);
```

Comment:
location is used for goto definition and should be location of the base class, i.e. where does 'goto def' navigates from `super`? `ILocatedMember` may need to be beefed up to hold multiple locations where there are multiple bases or new interface created and handled in the definition source.

Reply:
I see thanks

Reply:
I'll tried grabbing the next class in the mro list and using its location.. but in the editor gotodefinition still isnt' working. I just added a test to investigate more

Reply:
What are you goto def-ing on? `super()`, or its return value? Because `super` itself is specialized, it won't end up going anywhere. Its return value's goto-def might not do what you expect either, since assigning it away means that it's "defined" at the assignment.

Reply:
Next class may not work, if it is, say `Generic` of a sort. You may try to enumerate and fetch first that can be navigated to like `ILocatedMember.Location.IsValid`.

Reply:
ahh i see Jake,  Mikhail I'll see what happens with Generic

Reply:
I think i dont want goto definition to work on super() itself. 

I'm going to focus on the cases of goto on what comes after super(). ie super().func() and super.x